README.md

```diff
@@ -29,6 +29,7 @@ We use the GSoC project size parameters for estimating the expected time complex
     - [ABI/Layout handling for the automatic differentiation feature](#abilayout-handling-for-the-automatic-differentiation-feature)
     - [Improving parallel frontend](#improving-parallel-frontend)
     - [C codegen backend for rustc](#C-codegen-backend-for-rustc)
+    - [Prepare for publishing `stable_mir` crate](#prepare-for-publishing-stable_mir-crate)
```

Comment:
```suggestion
    - [Prepare `stable_mir` crate for publishing](#prepare-stable_mir-crate-for-publishing)
```
I think that's a more "natural sounding" :) Also I would put it above the C codegen backend, that's kind of a backup, so we want to keep it last in the compiler list.